runtime/pom.xml

```diff
@@ -73,6 +73,12 @@
             <artifactId>quarkus-hibernate-validator</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.thoughtworks.paranamer</groupId>
+            <artifactId>paranamer</artifactId>
```

Comment:
Maybe I'm not very eagle-eyed but I don't see any usages of this library in code.